sdk/resourcemanager/Proto.Client/authorization/lib/Azure.ResourceManager.Authorization.xml

```diff
@@ -0,0 +1,1804 @@
+<?xml version="1.0"?>
```

Comment:
We should not be checking in binaries into our repo. We should either commit the source and build this or pull it from a package manager.